### PR TITLE
remove examples

### PR DESCRIPTION
**Pull request by lynnco**
ref https://github.com/SexualHealthInnovations/callisto/issues/630

### CODE REVIEW

**Comment by kelseyq** [comment]
how does this change display?

**Comment by lynnco** [comment]
after

image

**Comment by lynnco** [comment]
before

image

File: wizard_builder/forms.py [hunk 1]
```diff
@@ -22,7 +22,7 @@ def __init__(self, *args, **kwargs):
         for item in self.items:
             question_id = 'question_%s' % item.pk
             self.fields[question_id] = item.make_field()
-            self.fields[question_id].help_text = mark_safe(item.descriptive_text + self.fields[question_id].help_text)
+            self.fields[question_id].help_text = mark_safe(item.example + '<br/><br/>' + item.descriptive_text + self.fields[question_id].help_text)
```
Comment (kelseyq) [inline wizard_builder/forms.py]: does anything still have example? i thought we tried to get rid of it. example > description > field doesn't seem like the right order if not

**Comment by lynnco** [comment]
That's just the content, the bigger error was that the example was being displayed in the page title https://github.com/SexualHealthInnovations/callisto/issues/630#issuecomment-271746368

File: wizard_builder/forms.py (hunk 1 above)
Comment (lynnco) [inline wizard_builder/forms.py]: its excluded on some fields, so yea it does look like you tried to get rid of it. It's still present on `When did it happen?` though, for whatever content set I'm working off of right now (either Canisius or DU)

Comment (lynnco) [inline wizard_builder/forms.py]: And yea I agree that doesn't seem like the right order. It was the currently existing order though. Should I change it from that?

**Comment by kelseyq** [comment]
I think we should get rid of the example field entirely.

**Comment by lynnco** [comment]
flake8 issue wasn't showing up on local b/c of a version mismatch :p

**Comment by lynnco** [comment]
👍 done

**Pull request merged**